src/components/Home/Sponsors.jsx

```diff
@@ -42,14 +42,14 @@ const Sponsors = ({ strings }) => (
         target="_blank"
         rel="noopener noreferrer"
       >
-        <img src="/assets/images/vp-logo.png" alt="" />
+        <img src="/assets/images/vp-logo.png" alt="VP Game home" />
       </a>
       <a
         href="https://www.openai.com/"
         target="_blank"
         rel="noopener noreferrer"
       >
-        <img src="/assets/images/openai-logo.png" alt="" />
+        <img src="/assets/images/openai-logo.png" alt="Open AI home" />
```

Comment:
Without the alt text, screen readers will read off the href very quickly, which may be challenging for users with cognitive disabilities. Providing alt text that reads a little more semantically helps reduce cognitive overhead.